include/method_enums.hpp

```diff
@@ -8,3 +8,11 @@ enum clustering_methods
     self_balancing_binary_tree = 2,
     candidate_selection_based_on_voting = 3
 };
+
+//!\brief An enum for the different refinement methods.
+enum refinement_methods
+{
+    no_refinement = 0,
+    sViper = 1,
+    sVirl_refinement_method = 2
```

Comment:
Small thing, we should say `sViper_refinement_method` to keep it consistent with the naming.